lightning/src/ln/onion_utils.rs

```diff
@@ -2172,6 +2268,156 @@ impl_writeable!(AttributionData, {
 	hmacs
 });
 
+impl AttributionData {
```

Comment:
Really nice comments here 🤩

Reply:
Thank you! This is what we were used to 😉